assets/styles/themes/authoring/default.scss

```diff
@@ -496,10 +496,10 @@ html.authoring.dark {
   .editorToolbar {
     border-top: 1px solid $body-color;
     border-bottom: 1px solid $body-color;
-    background: $body-bg;
+    background: $gray-400;
```

Comment:
Before, these menus, in dark-mode, were very difficult for me to see with black on dark-grey.

Before:
![image](https://user-images.githubusercontent.com/333265/179021698-54ed3749-8da6-47f8-bd02-543465be8121.png)

After:
![image](https://user-images.githubusercontent.com/333265/179021501-bee1fe45-f586-4839-a6a0-8ec99464b852.png)